sass.py

```diff
@@ -225,7 +225,9 @@ def _raise(e):
 
 def compile_dirname(
     search_path, output_path, output_style, source_comments, include_paths,
-    precision, custom_functions, importers,
+    precision, indented, custom_functions, importers,
+    source_map_contents=False, source_map_embed=False,
+    omit_source_map_url=False, source_map_root=None,
```

Comment:
despite being named without an underscore, `comple_dirname` is not a public interface so you can make all of these arguments required and non-optional